app/views/events/show.html.erb

```diff
@@ -1,5 +1,12 @@
-<h2>Event details</h2>
+<div class="ml-5">
+<h2 class="font-bold text-4xl">Event details</h2>
+
 <%= render 'event', event: @event %>
-<div style="margin-bottom: 10px"><%= link_to 'Edit event', edit_event_path(@event) %></div>
-<%= button_to 'Delete event', event_path(@event), style: "margin-bottom: 10px", 
-method: :delete, data: {turbo_confirm: "Are you sure?"} %> <%= link_to 'Back', events_path %>
+
+<%= button_to 'Edit event', edit_event_path(@event), method: :get, class: 'btn-1' %>
+<div class="">
+<%= button_to 'Delete event', event_path(@event), class: 'btn-2', 
+method: :delete, data: {turbo_confirm: "Are you sure?"} %> 
+</div>
+<%= button_to 'Back', events_path, method: :get, class: 'btn-3' %>
+</div>
```

Comment:
Add an blank line at the end of your code

Reply:
Done!